honeybadgermpc/preprocessing.py

```diff
@@ -22,17 +23,20 @@ class PreProcessingConstants(object):
     ONE_MINUS_ONE_FILE_NAME_PREFIX = f"{SHARED_DATA_DIR}one_minus_one"
     DOUBLE_SHARES_FILE_NAME_PREFIX = f"{SHARED_DATA_DIR}double_shares"
     READY_FILE_NAME = f"{SHARED_DATA_DIR}READY"
+    SHARE_BITS_FILE_NAME_PREFIX = f"{SHARED_DATA_DIR}share_bits"
```

Comment:
`ONE_MINUS_ONE` is also really the same as `BIT` except for a small local operation, I think we should only have one or the other